test/test_node_handler.py

```diff
@@ -367,6 +378,7 @@ def test_get_root_node_endpoint(mock_db_find_by_id, mock_init_sub_id):
             'name',
             'parent',
             'revision',
+            'result',
```

Comment:
I think it's a bit better to keep following the alphabetical order:
```suggestion
            'result',
            'revision',
```

Reply:
Updated.